src/EditorFeatures/Core.Wpf/Options/EditorConfigDocumentOptionsProviderFactory.cs

```diff
@@ -3,15 +3,25 @@
 using System;
 using System.ComponentModel.Composition;
 using Microsoft.CodeAnalysis.Options;
+using Microsoft.VisualStudio.CodingConventions;
 
 namespace Microsoft.CodeAnalysis.Editor.Options
 {
     [Export(typeof(IDocumentOptionsProviderFactory))]
     class EditorConfigDocumentOptionsProviderFactory : IDocumentOptionsProviderFactory
     {
+        private readonly ICodingConventionsManager _codingConventionsManager;
+
+        [ImportingConstructor]
+        [Obsolete("Never call this directly")]
```

Comment:
:bulb: A constant is now available for this string:

https://github.com/dotnet/roslyn/blob/990fbbbf1e29eca1b7e73aa755c446ce2858446d/src/Workspaces/Core/Portable/Workspace/Host/Mef/MefConstruction.cs#L7

❕ Need to include `error = true` on this attribute as well to prevent use outside of MEF.